src/geometry/resize.ts

```diff
@@ -53,36 +49,26 @@ export interface ResizeOptions {
  * @returns The new image.
  */
 export function resize(image: Image, options: ResizeOptions): Image {
+  const { interpolationType = 'bilinear' } = options;
   const {
-    interpolationType = 'bilinear',
-    borderType = 'constant',
+    borderType = interpolationType === 'bilinear' ? 'replicate' : 'constant',
```

Comment:
Why? Is it OpenCV behavior?

Reply:
I don't know but bilinear was better with replicate by default.

Reply:
This is probably not the right way to fix it.